### PR TITLE
Updated/completed HTML/CSS Youtube clone first project by Uchenna Martins Anya and Iris Enaelle Kineza 

### PR DESCRIPTION
**Pull request by KinezaEnaelle**
This is a project of HTML5 and CSS3  about cloning the Youtube video player page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        <link rel="shortcut icon" href="https://www.youtube.com/s/desktop/9f6471b0/img/favicon.ico" type="image/x-icon" >
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,900&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <header>
+            <nav>
+                <i class="fa fa-bars fa-lg"></i>
+                <div class="col logo">
+                    
+                    <img src="assets/images/YouTube-Logo.png" class="logo">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute, except under certain conditions

File: index.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                    <h3>Contributing to Open Source Part I: The Easy Way</h3>
+                    <p>60,463 views  •Feb 27, 2014</p>
+                    </div>
+                    
+                 </div>
+                 <hr>
+                 <div class="subscribe">
+                    <p>
+                        <button class="button">Subscribe</button>
+                        <img src="assets/images/7_avatar-512.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute, except under certain conditions

File: index.html [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+                         The Odin Project
+                        <p> 1.02K subscribers </p>
+                         
+                    </p>
```
Comment (arslanbisharat) [inline index.html:68]: No p element in scope but a p end tag is seen. Please remove this p tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+            
+                   <p>
+                       How do you contribute to open source?  In this video, 
+                       I'll describe the easiest way to make a simple contribution to an open source project 
+                       -- without using the command line, a text editor, "forking a repo", or even touching Git outside of Github. 
+                       I'll also describe what kinds of contributions open source software projects are typically looking for. 
+                       As an example, I use submitting a student solution to a web development exercise from The Odin Project, 
+                       a free online curriculum for learning web development with Ruby on Rails. 
+                       You can find more information about that project at <a href="http://theodinproject.com">http://theodinproject.com</a> <p>SHOW MORE</p>
+                   </p>
```
Comment (arslanbisharat) [inline index.html:79]: No p element in scope but a p end tag is seen. Please remove this p tag

File: index.html [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+                 
+                </div>
+                <hr>
+                <div class="comment">
+                    17 Comments &nbsp; &nbsp; <img src="assets/images/sort.png" class="sort"> &nbsp; &nbsp; SORT BY
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute, except under certain conditions

File: index.html [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+                </div>
+
+               <div class="add-comment">
+                   <img src="assets/images/7_avatar-512.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute, except under certain conditions

File: index.html [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+                   <textarea name="" id="" class="add-comment" placeholder="Add a public comment..."></textarea>
```
Comment (arslanbisharat) [inline index.html]: Bad value for attribute name on element Textarea. It must not be empty.

Comment (arslanbisharat) [reply index.html]: An ID must not be the empty string.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html>
```
Comment (arslanbisharat) [inline index.html]: Consider adding a lang attribute to the html start tag to declare the language of this document

File: index.html [hunk 9]
```diff
@@ -0,0 +1,206 @@
...
+                </div>
+                 <!-- Main video ends here -->
+
+                <div class="description" style="display: block;">
+                    <div style="float: right;">
+                        <i class="fa fa-thumbs-o-up"></i>
+                        <i class="fa fa-thumbs-down"></i>
+                        <i class="fa fa-share"> SHARE </i>
+                        <i class="fa fa-save"> SAVE</i>
+                        <!-- <img src="assets/images/save icon.png" class="save"> SAVE -->
```
Comment (arslanbisharat) [inline index.html]: Commented code should be deleted

File: assets/css/style.css [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+    margin-right: 2rem;
+}
+.col1 iframe {
+    width: 100%;
+    height: 400px;
+}
+main .col2 {
+    float: 5;
+    /* display: flex; */
+    /* border: 1px solid yellow; */
```
Comment (arslanbisharat) [inline assets/css/style.css]: Commented code should be deleted

File: assets/css/style.css [hunk 11]
```diff
@@ -0,0 +1,188 @@
...
+nav input {
+    width: 80%;
+    border: .5px solid #cccccc;
+    padding: .5rem 1rem;
+    /* margin-right: auto; */
+    right: 0;
+}
+nav button {
+    width: 10%;
+    /* margin-left: auto; */
```
Comment (arslanbisharat) [inline assets/css/style.css]: Commented code should be deleted

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request Changes
**Hello Team! Welcome to Microverse. This is an amazing place where you are going to learn many new things.** 

The above heading might scare you but don't worry your project is completed in the coding sense but here at Microverse we just not only learn how to code. We also learn how to become professional and show our work to others. I saw your code. That's amazing. you have met all the minimum requirements to approve this project but  I am sorry to inform you that you didn't follow every instruction. You can have a look[ here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/)
I can see you worked hard on this project but I am sorry to inform you that I cannot Approve this project because it is not completed yet.😟
_It's okay. We are here to learn. Don't get discouraged. Work more hard._
**Please have a look at the suggestions to get more understanding. 👀**

- [x] Your PR is not descriptive Please tell us what you did in this project dhow you did. Is there anything else you want to tell TSEs about this project?
- [x] we cannot have empty attributes please resolve those issues. 
- [x]  There are many closing p tags which re not being used please remove them. 
- [x] Commented code should be deleted
- [x] Search button should be aligned with the search bar. Please remove space between them
- [x] Add live version of your project in the readme file. You can easily get one by pasting the URL of your index.html file into this [website](https://raw.githack.com/)
- [x] **Optional** Add this to assests
chrome-capture (44)


- [x] **Optional** Please remove the space so that the link works on Readme
chrome-capture (42)

- [x] **Optional** Please add a brief description of your project in the About section
chrome-capture (43)

**Please make the changes and request re-review for the given project**
Stay safe😇

### Happy Coding
-------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by jubaan — APPROVED** [review]
# :tada::tada: Your project has been approved!!! :fireworks::fireworks:

## :wave: :wave: :wave: Hello, team! 

Great job working on the requested changes! :clap::clap::clap::clap: 

- [x] :100: Your PR is not descriptive Please tell us what you did in this project dhow you did. Is there anything else you want to tell TSEs about this project?
- [x] :100: We cannot have empty attributes please resolve those issues.
- [x] :100: There are many closing p tags which re not being used please remove them.
- [x] :100: Commented code should be deleted
- [x] :100: Search button should be aligned with the search bar. Please remove space between them
- [x] :100: Add live version of your project in the readme file. You can easily get one by pasting the URL of your index.html file into this website

### [OPTIONAL] 
- For future projects use more descriptive feature branch names. Example:
:-1: ```feature-branch```. Makes the reader question about What is this branch about? Whats the feature?
:+1: :heavy_check_mark:  ```feature-google-clone```. Descriptive, short, meaningful, and more readable
- Commit messages should be more informative. This will build a better history for future readers or yourself if you want to understand the decisions made in the past.
:-1: ```Update```
:+1: ```Update README contact info```

**Pull request merged**